integration-tests/kogito-maven/src/test/java/io/quarkus/kogito/maven/it/KogitoDevModeIT.java

```diff
@@ -15,7 +15,7 @@
     @Test
     public void testThatTheKogitoApplicationRuns() throws MavenInvocationException, IOException {
         testDir = initProject("projects/simple-kogito", "projects/project-classic-run-kogito");
-        run();
+        run("-e");
```

Comment:
Do we want to keep that?

Reply:
Yes IMO.  It doesn't do any harm, and you always want to add it when things go wrong.